File: app/src/main/java/ru/vincetti/vimoney/data/models/AccountListModel.java
package ru.vincetti.vimoney.data.models;

import androidx.room.ColumnInfo;

public class AccountListModel {

    public int id;
    public String name;
    public String type;
    public int sum;
    @ColumnInfo(name = "account_symbol")
    public String curSymbol;
    public String color;
    @ColumnInfo(name = "archive")
    public boolean isArhive;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public int getSum() {
        return sum;
    }

    public boolean isArhive() {
        return isArhive;
    }

    public String getSymbol() {
        return curSymbol;
    }

    public String getColor() {
        return color;
    }
}

File: app/src/main/java/ru/vincetti/vimoney/data/models/AccountListModel.kt
package ru.vincetti.vimoney.data.models

import androidx.room.ColumnInfo

data class AccountListModel(

    val id: Int,

    val name: String,

    val type: String,

    val sum: Int,

    @ColumnInfo(name = "symbol")
    val curSymbol: String = CurrencyModel.DEFAULT_CURRENCY_SYMBOL,

    val color: String,

    @ColumnInfo(name = "archive")
    val isArchive: Boolean,

    var currency: Int = AccountModel.DEFAULT_CURRENCY,

    @ColumnInfo(name = "extra_key")
    var extraKey: String = "",

    @ColumnInfo(name = "extra_value")
    var extraValue: String = "",

    @ColumnInfo(name = "need_all_balance")
    var needAllBalance: Boolean = true,

    @ColumnInfo(name = "need_on_main_screen")
    var needOnMain: Boolean = true
)
